gym/lib/gym/error_handler.rb

```diff
@@ -153,6 +153,7 @@ def print_full_log_path
 
       def print_xcode9_plist_warning
         return unless Helper.xcode_at_least?("9.0")
+        return unless Gym.config[:export_options].kind_of?(Hash)
```

Comment:
Could you explain the context on why this new line is necessary?

Reply:
Hi, It's to prevent crash in case of packaging error AND if you have set export_options to a path.

Indeed, the line `provisioning_profiles = export_options[:provisioningProfiles] || []` will triggered a `[!] no implicit conversion of Symbol into Integer (TypeError)`

Should I add that in comment ?

Reply:
It would be really great if you could add that comment. It helps us understand the one-offs in the codebase since it's not super common to accept multiple types as args.